lib/langchain/llm/google_gemini.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require "active_support/core_ext/hash/keys"
+require "active_support/core_ext/string/inflections"
```

Comment:
Do we need to use `active_support`? I specifically [removed it](https://github.com/patterns-ai-core/langchainrb/pull/639) to reduce the number of dependencies.

Reply:
Two utilities have been used. I am not much familiar with Ruby. if there's another way to implement that, I'm open to suggestions

Reply:
I've created internal utils class